counterpartylib/lib/messages/issuance.py

```diff
@@ -164,7 +210,8 @@ def validate (db, source, destination, asset, quantity, divisible, callable_, ca
     if util.enabled('integer_overflow_fix', block_index=block_index) and (fee > config.MAX_INT or quantity > config.MAX_INT):
         problems.append('integer overflow')
 
-    return call_date, call_price, problems, fee, description, divisible, reissuance
+    return call_date, call_price, problems, fee, description, divisible, reissuance, reissued_asset_longname
```

Comment:
i'm not convinced that you need to return `reissued_asset_longname`. if `reissuance == True`, then `subasset_longname == reissued_asset_longname` and you can just use the `subasset_longname` passed in. if `reissuance == False`, then `reissued_asset_longname is None` and you can just use `reissuance`